cmd/mapcidr/main.go

```diff
@@ -289,20 +282,36 @@ func prepareIPsFromCidrFlagList(items []string) []string {
 func process(wg *sync.WaitGroup, chancidr, outputchan chan string) {
 	defer wg.Done()
 	var (
-		allCidrs []*net.IPNet
-		pCidr    *net.IPNet
-		ranger   *ipranger.IPRanger
-		err      error
-		hasSort  = options.SortAscending || options.SortDescending
-		ipRange  []net.IP
+		allCidrs     []*net.IPNet
+		pCidr        *net.IPNet
+		ranger       *ipranger.IPRanger
+		err          error
+		hasSort      = options.SortAscending || options.SortDescending
+		ipRangeMap   = make(map[int][]net.IP)
```

Comment:
Why do we need a map, why can't the array work? Why do you need iprangeCount also?

Reply:
Removed the map with the list